os/mm/umm_heap/umm_realloc.c

```diff
@@ -137,7 +137,9 @@ void *realloc_at(int heap_index, void *oldmem, size_t size)
 
 FAR void *realloc(FAR void *oldmem, size_t size)
 {
-	int prev_heap_idx;
+	#ifndef CONFIG_APP_BINARY_SEPARATION
```

Comment:
This alignment is wrong.. you should keep #if indicator to leftmost, please check other place usage of this...
Also, I'm not sure whether #ifndef is default usage. Normally, it should be either `#ifdef` or `#if !defined(CONFIG_XXX..)`
@sunghan-chang  could you please help to clarify?

Reply:
Agree two things @edwakuwaku said.